release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -475,6 +475,61 @@ For more information, see xref:../registry/configuring_registry_storage/configur
+==== Automatic deletion of resources for GCP Filestore is generally available
+In earlier versions of {product-title}, when destroying a cluster, Google Compute Platform (GCP) Filestore Storage did not delete all of the cloud resources belonging to that cluster. This required manually deleting all of the persistent volume claims (PVCs) that used the Filestore storage class before destroying the cluster.
+
+With {product-title} 4.17, when destroying a cluster the {product-title} the installer should generally delete all of the cloud resources that belong to that cluster, and therefore manual deletion of PVCs should not be required. However, due to the special nature of the Google Compute Platform (GCP) Filestore resources, the automated cleanup process might not remove all of resources in some cases. This feature is supported as generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc[Google Compute Platform Filestore CSI Driver Operator].
+
+[id="ocp-4-17-storage-azure-file-snapshot"]
+==== Azure File CSI supports snapshots (Technology Preview)
+{product-title} 4.17 introduces volume snapshot support for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator. This capability is supported as a Technology Preview feature.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi.adoc#csi-drivers-supported_persistent-storage-csi[CSI drivers supported by {product-title}] and xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc[CSI volume snapshots].
+
+[id="ocp-4-17-storage-multi-vcenter-support"]
+==== Multiple vCenter support for vSphere CSI (Technology Preview)
+{product-title} v4.17 introduces the ability to deploy {product-title} across multiple vSphere clusters (vCenters). This feature is supported with Technology Preview status.
+
+Multiple vCenters can only be configured during installation. The maximum number of supported vCenter clusters is three.
+
+// For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-17-storage-selinuxmount-dev-preview"]
+==== RWX/RWO SELinux Mount (Developer Preview)
+Pods may take a very long time to start when the volume contains a large number of files. To avoid SELinux labeling issues while keeping SELinux confining, you can enable the ReadWriteMany/ReadWriteOnce (RWX/RWO) SELinux Mount feature. Be advised that the RWX/RWO SELinux Mount feature is a Developer Preview feature. It is not supported by Red Hat, and you should not enable this feature set on production or clusters that you plan to maintain over time.
```

Comment:
```suggestion
Pods may take a very long time to start when the volume contains a large number of files. To avoid SELinux labeling issues while keeping SELinux confining, you can enable the ReadWriteMany/ReadWriteOnce (RWX/RWO) SELinux Mount feature. Be advised that the RWX/RWO SELinux Mount feature is a Developer Preview feature. It is not supported by Red{nbsp}Hat, and you should not enable this feature set on production clusters that you plan to maintain over time.
```